package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jupyterlab-topbar-extensions",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Temporary fix for https://github.com/jupyterlab-contrib/jupyterlab-topbar/issues/114